types/jquery/index.d.ts

```diff
@@ -30846,6 +30852,7 @@ $( ".box" ).on( "click", "button", function( event ) {
 ```
          */
         delegateTarget: TTarget;
+        handleObj: HandleObject<TTarget, TData>;
```

Comment:
This addition of this line causes a lot of errors similar to

```
Argument of type 'JQuery<Text>' is not assignable to parameter of type 'string | Node | Node[] | JQuery<Node>'.
  Type 'JQuery<Text>' is not assignable to type 'JQuery<Node>'.
```

at

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5355d29def7efae47ac70460b559b59a27551e7b/types/jquery/jquery-tests.ts#L5724

where `.insertBefore()` is declared as

```typescript
    insertBefore(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Node> | JQuery<Node>): this;
```

I'm not sure I understand why assignability changes from adding `handleObj`.

Reply:
@andy-ms any insight into why this happens?

I know I managed to work around it for this PR but I'm running into the same set of errors in a seemingly unrelated change in another PR I'm working on.